pennylane/resources/second_quantization.py

```diff
@@ -36,7 +79,37 @@ class SQ:
         br (int): number of bits for ancilla qubit rotation
         alpha (int): number of bits for the keep register
         beta (int): number of bits for the rotation angles
+
+    .. details::
+        :title: Usage Details
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane import numpy as np
+
+            symbols  = ['H', 'H', 'O']
+            geometry = np.array([[0.00000000,  0.00000000,  0.28377432],
+                                 [0.00000000,  1.45278171, -1.00662237],
+                                 [0.00000000, -1.45278171, -1.00662237]], requires_grad = False)
+
+            mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
+
+            core_, one, two_ph = qml.qchem.electron_integrals(mol)()
+            two = np.swapaxes(two_ph, 1, 3) # convert to chemist notation
```

Comment:
This is the part of the UI that I don't like. Let's keep in mind if there is anything we can do in the future to avoid this